src/main/java/edu/hm/hafner/util/PathUtil.java

```diff
@@ -259,7 +259,7 @@ public String createAbsolutePath(final @Nullable String directory, final String
      *
      * @return {@code true} if this path is an absolute path, {@code false} if a relative path
      */
-    public boolean isAbsolute(final String fileName) {
+    public boolean isAbsolute( String fileName) {
         return FilenameUtils.getPrefixLength(fileName) > 0;
```

Comment:
checkstyle error hopefully
```suggestion
        return   FilenameUtils.getPrefixLength(
        fileName) > 
        0;
```